drltrace/drltrace_win.config

```diff
@@ -1168,4 +1168,5 @@ wchar*|lstrcpyW|__out wchar*|wchar*
 char*|lstrcpynA|__out char*|char*|int
 wchar*|lstrcpynW|__out wchar*|wchar*|int
 int|lstrlenA|char*
+
```

Comment:
I meant a newline at the end of the last line: seems weird to split off one line, right?